QuestionAnswering/README.md

```diff
@@ -0,0 +1,54 @@
+# Question Answering on Android with Kotlin
+
+## Introduction
+
+Question Answering (QA) is one of the common and challenging Natural Language Processing tasks. With the revolutionary transformed-based [Bert](https://arxiv.org/abs/1810.04805) model coming out in October 2018, question answering models have reached their state of art accuracy by fine-tuning Bert-like models on QA datasets such as [Squad](https://rajpurkar.github.io/SQuAD-explorer). [Huggingface](https://huggingface.co)'s [DistilBert](https://huggingface.co/transformers/model_doc/distilbert.html) is a smaller and faster version of BERT - DistilBert "has 40% less parameters than bert-base-uncased, runs 60% faster while preserving over 95% of BERT’s performances as measured on the GLUE language understanding benchmark."
+
+In this demo app, written in Kotlin, we'll show how to quantize and convert the Huggingface's DistilBert QA model to TorchScript and how to use the scripted model on an Android demo app to perform question answering.
+
+## Prerequisites
+
+* PyTorch 1.7 or later (Optional)
+* Python 3.8 (Optional)
+* Android Pytorch library 1.7 or later
```

Comment:
Specify the version of pytorch and pytorch_android that was used for model that can be downloaded